mantle/platform/qemu.go

```diff
@@ -1028,9 +1028,11 @@ func (builder *QemuBuilder) Exec() (*QemuInstance, error) {
 	if builder.ConfigFile != "" && !builder.configInjected {
 		if builder.supportsFwCfg() {
 			builder.Append("-fw_cfg", "name=opt/com.coreos/config,file="+builder.ConfigFile)
-		} else if builder.primaryDisk == nil {
-			// Otherwise, we should have handled it in builder.AddPrimaryDisk
-			panic("Ignition specified but no primary disk")
+		} else {
+			// Alternative to fw_cfg, should be generally usable on all arches,
+			// especially those without fw_cfg support.
```

Comment:
Note it's technically not usable yet on arches that do support `fw_cfg` (see https://github.com/coreos/ignition/issues/928#issuecomment-592732795). But meh... not worth a respin.

Reply:
Oh... sorry bad wording on my side, should have been possibly...., will keep it on back of my mind and will fix it in some future PR.